### PR TITLE
[NodeJS][25.bot-logging] - Fix minor typos and improve README structure and content

### PR DESCRIPTION
**Pull request by gasper-az**
## Proposed Changes
- Add 'appId' and 'appSecret' to msbot command and instructions to obtain them
- Update botbuilder version from 4.0.6 to 4.1.5 to solve vulnerabilities issues
- Update README.md file to match with template
- Fix minor typos

### CODE REVIEW

File: samples/javascript_nodejs/25.logger-bot/README.md [hunk 1]
```diff
@@ -7,15 +7,24 @@ In this example, the bot creates a new folder in the project directory called `l
 
 This bot example uses [restify](https://www.npmjs.com/package/restify).
 
-# To run the bot
+# To try this sample
+- Clone the repository
+    ```bash
+    git clone https://github.com/microsoft/botbuilder-samples.git
+    ```
+- In a terminal, navigate to samples/javascript_nodejs/25.logger-bot
```
Comment (joseques) [inline samples/javascript_nodejs/25.logger-bot/README.md]: Please use `code style` when mentioning paths, files or variable names.

File: samples/javascript_nodejs/25.logger-bot/README.md [hunk 2]
```diff
@@ -7,15 +7,24 @@ In this example, the bot creates a new folder in the project directory called `l
...
+    ```bash
+    cd samples/javascript_nodejs/25.logger-bot
+    ```
+- [Optional] Update the .env file under samples/javascript_nodejs/25.logger-bot with your botFileSecret For Azure Bot Service bots, you can find the botFileSecret under application settings.
```
Comment (joseques) [inline samples/javascript_nodejs/25.logger-bot/README.md]: Please use code style when mentioning paths, files or variable names.

File: samples/javascript_nodejs/25.logger-bot/README.md [hunk 3]
```diff
@@ -42,18 +51,13 @@ In this example, the bot's state is used to track number of messages.
...
 You can use the [MSBot][5] Bot Builder CLI tool to clone and configure the services this sample depends on.
 
-To install all Bot Builder tools -
-
-Ensure you have [Node.js](https://nodejs.org/) version 8.5 or higher
-
-```bash
-npm i -g msbot chatdown ludown qnamaker luis-apis botdispatch luisgen
-```
+To install all Bot Builder tools, follow these [instructions][11].
```
Comment (joseques) [inline samples/javascript_nodejs/25.logger-bot/README.md]: For consistency, can you change the instructions as we have it in [other samples](https://github.com/southworkscom/BotBuilder-Samples/commit/99f4189f5330c9d61dc76a2effaa3cc41d97d0f7#diff-671f2784abcf5c38d62133d9f710dbaaR41)?

**Review by joseques — CHANGES_REQUESTED** [review]

**Review by VictorGrycuk — APPROVED** [review]
Fix @joseques requests and 🚢 it

File: samples/javascript_nodejs/25.logger-bot/README.md [hunk 4]
```diff
@@ -40,20 +49,13 @@ In this example, the bot's state is used to track number of messages.
...
-Ensure you have [Node.js](https://nodejs.org/) version 8.5 or higher
-
-```bash
-npm i -g msbot chatdown ludown qnamaker luis-apis botdispatch luisgen
-```
+You can use the [MSBot][5] Bot Builder CLI tool to clone and configure any services this sample depends on. In order to install this and other tools, you can read [Installing CLI Tools][11]
 
 To clone this bot, run
+```bash
+msbot clone services -f deploymentScripts/msbotClone -n <BOT-NAME> -l <Azure-location> --subscriptionId <Azure-subscription-id> --appID <App-Id> --appSecret <App-Secret>
```
Comment (dfavretto) [inline samples/javascript_nodejs/25.logger-bot/README.md]: Be careful here, the `--appID` should be `--appId`
```suggestion
msbot clone services -f deploymentScripts/msbotClone -n <BOT-NAME> -l <Azure-location> --subscriptionId <Azure-subscription-id> --appId <App-Id> --appSecret <App-Secret>
```

File: samples/javascript_nodejs/25.logger-bot/README.md [hunk 5]
```diff
@@ -72,3 +74,5 @@ msbot clone services -f deploymentScripts/msbotClone -n myChatBot -l <Azure-loca
 [8]: https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-v4-storage?view=azure-bot-service-4.0&tabs=jsechoproperty%2Ccsetagoverwrite%2Ccsetag
 [9]: https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-v4-state?view=azure-bot-service-4.0&tabs=js
 [10]: https://dev.botframework.com
+[11]: https://github.com/southworkscom/BotBuilder-Samples/blob/master/Installing_CLI_tools.md#install-cli-tools
```
Comment (dfavretto) [inline samples/javascript_nodejs/25.logger-bot/README.md]: As you can see [here](https://github.com/Microsoft/BotBuilder-Samples/blame/master/samples/javascript_nodejs/06.using-cards/README.md#L47) we tend to use the relative path to the `Install CLI Tools` file, taking advantage it's in the root folder of the repo, and the user has to clone the entire repo to try the sample.
```suggestion
[11]: ../../../Installing_CLI_tools.md
```

**Review by dfavretto — CHANGES_REQUESTED** [review]
Along with the inline requested changes, I suggest to re-write the PR using Present conjugations instead of Past conjugations, because in Past time, the one reading the PR may interpret the changes are already implemented, but the idea is to make the other person understand that the PR is implementing these changes in the moment they read it. 

Also, try making the title more generic, not going to the specific change in the `msbot clone services` command, it should be something more like `Fix typos and improve Readme structure and content`

**Review by dfavretto — APPROVED** [review]

**Review by joseques — APPROVED** [review]

**Comment by dfavretto** [comment]
Closed due to Microsoft#870

**Pull request closed without merging**